packages/terra-slider/translations/en-US.json

```diff
@@ -0,0 +1,3 @@
+{
+  "Terra.slider.Slider": "Slider"
```

Comment:
Is there a JIRA logged to add the translations for other locales?

Reply:
There is no Jira logged for this currently, maybe because it was not a requirement for the passthrough. I agree that one should be logged and will follow-up on this and post here.

Reply:
Thank you @chrismichalewicz!

Reply:
Where would we be using this translation?